libnetwork/resolver_test.go

```diff
@@ -120,7 +120,7 @@ func TestDNSIPQuery(t *testing.T) {
 
 	w := new(tstwriter)
 	// the unit tests right now will focus on non-proxyed DNS requests
-	r := NewResolver(resolverIPSandbox, false, sb.Key(), sb.(*sandbox))
+	r := NewResolver(resolverIPSandbox, false, sb.(*sandbox))
```

Comment:
(Just a random comment); the `sb.(*sandbox)` is quite awkward here; guess it's needed, but 🤷‍♂️